packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1176,19 +1110,19 @@ class _DialState extends State<_Dial> with SingleTickerProviderStateMixin {
     TimeOfDay(hour: 11, minute: 0),
   ];
 
-  static const List<TimeOfDay> _pmHours = <TimeOfDay>[
+  static const List<TimeOfDay> _twentyFourHours = <TimeOfDay>[
     TimeOfDay(hour: 0, minute: 0),
```

Comment:
optional: you could replace this section with a for loop inside the list literal.

Reply:
Hmm I think I prefer it being more explicitly laid out so you can see the exact `TimeOfDay` objects being created